.gitattributes

```
@@ -0,0 +1,4 @@
+# get php syntax highlighting for these file extensions
+*.module linguist-language=php
+*.install linguist-language=php
+*.test linguist-language=php
```

Let's please add .profile to the list:

```suggestion
*.test linguist-language=php
*.profile linguist-language=php
```